recipes/jupyterlab_genv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterlab_genv" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_genv-{{ version }}.tar.gz
+  sha256: c850e8670f10e8c6926de1fce8b85de17ffdb69a6cccc3ac8cdc38fd6c4954f9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - hatchling >=1.3.1
+    - jupyterlab ~=3.1
+    - hatch-jupyter-builder >=0.5
+  run:
```

Comment:
Since this depends on genv, it should probably be listed here?

Reply:
it's a very good point!
I think you are right. 
added it!

Reply:
ok after adding `genv` as a dependency, the build failed a few times:

at first it didn't work on Windows as `genv` does not support Windows

when I skipped build on Windows, it said that:
> noarch packages can't have skips with selectors. If the selectors are necessary, please remove noarch: python.

when I removed the `noarch:python`, it said that:
> Non noarch packages should have python requirement without any version constraints

what do you think is the best solution here, should I specify the minimum version of Python?

Reply:
Just ignore the windows failure, that's fine – the package will just not be available there, since genv isn't either. The package is only built on linux but can be installed on all OS so it doesn't matter.

Reply:
> I'm also asking because what if genv is installed out of Conda?
specifying it here would require it to be installed in Conda even though it might be installed out of it?

Depends, if it's properly installed, conda will find it and just not install it.

Reply:
ok great so I restored the `noarch: python`, set `genv` as a dependency, and removed the `skip: true # [win]`
I'm pretty sure that the check would fail because of build on Windows fails, so I hope it is not a stopper for merging 👌

Reply:
hi @BastianZim 
pls let me know if it's fine now or any further changes should be done

thanks!